misc/prio3_multiproof_robustness.py

```diff
@@ -0,0 +1,100 @@
+# prio3_multiproof_robustness.py - Plot robustness bounds for various parameters.
+# Use `sage -python prio3_multiproof_robustness.py`
+import matplotlib.pyplot as plt
+import math
+
+FIELD128_MODULUS = 2**66 * 4611686018427387897 + 1  # Field128.MODULUS
+FIELD64_MODULUS = 2**32 * 4294967295 + 1            # Field64.MODULUS
+
+NUM_REPORTS = 1000000000
+
+
+def soundness(gadget_calls, gadget_degree, field_size):
+    '''
+    ia.cr/2019/188, Theorem 4.3
+
+    gadget_calls - number of times the gadget is called
+
+    gadget_degree - arithmetic degree of the gadget
+
+    field_size - size of the field
+    '''
+    return gadget_calls * gadget_degree / (field_size - gadget_calls)
+
+
+def robustness(epsilon, ro_queries, prep_queries, num_proofs):
+    '''
+    ia.cr/2023/130, Theorem 1, ignoring negligible terms and assuming the bound
```

Comment:
btw, I believe when `num_proofs` increase,  these negligible may become more significant since the first term is exponentially shrinking. Any harm to include it anyway?

Reply:
Sure, I can add it.